src/encoded/types/file.py

```diff
@@ -866,25 +866,28 @@ def _open_data_url(self, status, filename):
             return None
         if status in ['released', 'archived']:
             open_data_bucket = '4dn-open-data-public'
-            if 'wfoutput' in self.get_bucket(self.registry):
-                bucket_type = 'wfoutput'
-            else:
-                bucket_type = 'files'
+            bucket_type = 'wfoutput'  # almost always going to be wfoutput
             open_data_key = 'fourfront-webprod/{bucket_type}/{uuid}/{filename}'.format(
                 bucket_type=bucket_type, uuid=self.uuid, filename=filename,
             )
-            # Check if the file exists in the Open Data S3 bucket
+            extra_open_data_key = 'fourfront-webprod/{bucket_type}/{uuid}/{filename}'.format(
+                bucket_type='files', uuid=self.uuid, filename=filename,
+            )
+            # Check if the file exists in the Open Data S3 bucket under both wfoutput and files paths
             client = boto3.client('s3')
-            try:
+            for key in [open_data_key, extra_open_data_key]:
                 # If the file exists in the Open Data S3 bucket, client.head_object will succeed (not throw ClientError)
                 # Returning a valid S3 URL to the public url of the file
-                self._head_s3(client, open_data_bucket, open_data_key)
+                try:
+                    self._head_s3(client, open_data_bucket, open_data_key)
+                except ClientError:
+                    continue  # try the other key
                 location = 'https://{open_data_bucket}.s3.amazonaws.com/{open_data_key}'.format(
-                    open_data_bucket=open_data_bucket, open_data_key=open_data_key,
+                    open_data_bucket=open_data_bucket, open_data_key=key,
                 )
```

Comment:
I think (untested):
```
                location = f"https://{open_data_bucket}.s3.amazonaws.com/{key}"
```

Reply:
(I would also be OK renaming the iteration variable from `key` to `some_open_data_key` if you wanted to remind yourself that it was in fact such a key. But that's a personal taste thing.)